app/assets/src/components/common/CoverageVizBottomSidebar/CoverageVizBottomSidebar.jsx

```diff
@@ -467,13 +467,32 @@ export default class CoverageVizBottomSidebar extends React.Component {
 
     if (!this.isAccessionDataValid(currentAccessionData)) {
       return (
-        <div className={cs.unknownErrorContainer}>
-          <div className={cs.unknownErrorMessage}>
-            Failed to load data due to an unknown error. Please{" "}
-            <a className={cs.helpLink} href="mailto:help@idseq.net">
-              contact us
-            </a>{" "}
-            for help.
+        <div className={cs.noDataBody}>
+          <div className={cs.noDataContainer}>
+            <div className={cs.text}>
+              <div className={cs.message}>
+                Sorry, we failed to load the coverage data due to an unexpected
+                error.
+              </div>
+              <a
+                className={cs.linkWithArrow}
+                href="mailto:help@idseq.net"
+                onClick={() =>
+                  logAnalyticsEvent(
+                    "CoverageVizBottomSidebar_accession-data-invalid-contact-us_clicked",
+                    {
+                      accessionId: currentAccessionSummary.id,
+                      taxonId: params.taxonId,
+                      sampleId,
+                    }
+                  )
+                }
+              >
+                Contact us for help
+                <i className={cx("fa fa-chevron-right", cs.rightArrow)} />
```

Comment:
An `<i />` tag doesn't make any sense here - perhaps go with a `<span />`?

Reply:
Great point!

The <i> tag has been used by Font Awesome for icons. Example: https://fontawesome.com/v4.7.0/icon/tag.
There doesn't seem to be a great reason for it. https://stackoverflow.com/questions/21559816/why-does-font-awesome-recommend-using-i-elements-instead-of-span-elements-i

However, since we use <i> tags elsewhere in our app for icons, I think we should keep the <i> tag for consistency.